source/pkcore/commands.rst

```diff
@@ -314,7 +314,7 @@ Version
 | Usage: ``/bending version``
 | Aliases: ``[v]``
 
-Displays the version of the core plugin.
+Displays the version of the ProjectKorra Core jar installed on the server.
```

Comment:
Change `ProjectKorra Core jar installed on the server` to `ProjectKorra plugins installed on the server`